scripts/vmbuild.rb

```diff
@@ -172,7 +175,11 @@ def verify_run(output)
         $log.info `scp #{destination} #{FILE_SERVER_ACCOUNT}@#{FILE_SERVER}:#{file_rdu_dir.join(file_name)}`
       end
     end
+
+    # The final image is moved out of STORAGE_DIR at this point, delete all other files created during build
+    temp_file_uuid.each { |uuid| FileUtils.rm(Dir.glob("#{STORAGE_DIR}/#{uuid}.*"), :verbose => true) }
```

Comment:
Updated to use `rm_f`.

I can't use "#{STORAGE_DIR}/*" because some files might be still in use.  For example, if 2 nightly builds are running at the same time (this is possible because we check if a build is running by `virsh list` but the 1st nightly build might be running a part of the process that doesn't initiate virsh process) when 2nd one started...  So I just need to delete what's used in that particular build.